diagnosis/scripts/updateconfiguration.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash -e
+
+
+
+
+
+agentScript() {
+    cat <<EOF > agent.sh
+#!/bin/bash -e
+
+# cloud provider
+sudo cp /etc/kubernetes/azure.json /etc/kubernetes/azure.bak
+sudo jq '.cloudProviderRatelimit = true | .cloudProviderRateLimitQPS = 3 | .cloudProviderRateLimitBucket = 10 | .cloudProviderBackoffRetries = 1 | .cloudProviderBackoffDuration = 30' /etc/kubernetes/azure.bak | sudo dd status=none of=/etc/kubernetes/azure.json
+
+# kubelet
+sudo cp /etc/default/kubelet /etc/default/kubelet.bak
+sudo sed -i -e 's/--node-status-update-frequency=[0-9]*[a-z]/--node-status-update-frequency=1m/' /etc/default/kubelet
+
+# restart
+sudo systemctl daemon-reload
+sudo systemctl restart kubelet
+
+echo "=> azure.json updates"
+sudo grep -E 'cloudProviderRatelimit"|cloudProviderRateLimitQPS"|cloudProviderRateLimitBucket"' /etc/kubernetes/azure.json
```

Comment:
this line is not showing all the changes made to azure.json

Reply:
Updated.